### ISSUE TITLE
Get nan training loss

### ISSUE THREAD

**Issue by oOShauNOo** [mravanelli/pytorch-kaldi#32]
Hi, thanks for the great job! @mravanelli 
I am running the TIMIT_1dnn+5liGRU+1dnn_best config. But for my use setting, I changed the mfcc, fbank and fmllr framshift to 5ms, instead of the default setting 10ms. The problem is that the training loss and the valid loss are nan. BTW, I have successfully run your default setting, where frameshift is 10ms. Could you please suggest possible issues? Thank you.
<img width="344" alt="wx20181223-100038 2x" src="https://user-images.githubusercontent.com/40478651/50380165-9d47c480-0699-11e9-853c-d2b61573bc8f.png">

**Comment by mravanelli** [comment]
Hi,
if you change the shift in the feature computation you should also provide
labels with that shift. You thus have to recompute the alignements (i.e.,
the labels)  by running the gmm part of the s5 kaldi recipe again with the
new features and set the right aligment folder into the cfg file.

**Comment by mravanelli** [comment]
Please,
let me know the final performance. It's possible that you might have to
fine tune a little bit the hyperparameters since with you approach you have
to process more data (you have one new feature after 5 ms rather than
10ms).
